### DISCUSSION TITLE
개발자모드에서네이버지도  graphql JSON 이제는 빈 값만 나오는 듯 하네요?ㅠㅠ

### DISCUSSION THREAD

**Discussion by sosunnyproject** [changdoc/naver-vaccine-macro#57] [Q&A]
병원별 key 를 알아내려고 네이버 지도에서 병원들 여러개 떠있는 채로 새로고침하면서 graphql 데이터 계속 로드해봤습니다. 네이버에서 막은건지 graphql 에 계속 이렇게 `[{"data":{"shapes":[]}}]
` 빈  [ ] 값만 보이네요...  병원별 key 정보도 다 숨겼는지 찾을 수가 없어서, 알림 오면 그 병원예약 링크로 접속해서 새로고침하고 있습니다. 이 링크도 하루/n시간이 지나면 더이상 접속 불가한듯 하구요. 

혹시 지금 지도에서 병원리스트 GRAPHQL JSON 데이터 보이는 분들 있으신지 궁금합니다ㅠㅠ

**Comment by OneTop4458** [comment]
![image](https://user-images.githubusercontent.com/40881444/126584846-b2575f4b-5dc2-494d-a12e-9fa3e6527a9a.png)

정상적으로 잘 보이고 있습니다 Status 200 의 graphql 이 맞는지 확인 바라며

알림 톡으로 오는 링크의 역시 링크 주소 복사 후 다음과 같이 &due_date=20210721_2000 형태의 영역 지우시면 무제한 접속 가능합니다.
https://v-search.nid.naver.com/reservation/standby?orgCd=12398705&sid=1460891428&due_date=20210721_2000

**Reply by sosunnyproject** [reply]
오 감사합니다 ㅠ 크롬에서는 계속 빈 JSON만 보였는데, Brave 에서는 JSON 데이터 하나 건졌습니다! 
모바일에서 말고 PC 브라우저에서 이미 지난 알림 링크 열어서, due_date 부분을 지우니 예약신청 창이 잘 뜨네요! 너무 감사합니다~!

**Comment by changdoc** [comment]
크롬에 매크로 정상 설치되어 있다면 지난 링크 누르면 due_date는 지우고 바로 예약신청 페이지로 이동 하도록 조치가 되어 있긴합니다.
